custom_components/hyxi_cloud/switch.py

```
@@ -175,11 +176,8 @@ async def async_turn_on(self, **kwargs) -> None:
             self._attr_is_on = True
             self.async_write_ha_state()
             await self.coordinator.async_request_refresh()
-        except HyxiApiClient.ControlError as err:
-            _LOGGER.error(
-                "Failed to power on microinverter %s: %s", mask_sn(self._sn), err
-            )
-            raise
+        except Exception as err:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)